.infrastructure/06_alb.tf

```
@@ -0,0 +1,35 @@
+resource "aws_alb" "cms_load_balancer" {
+  name            = "cms-load-balancer"
+  subnets         = module.vpc.public_subnets
+  security_groups = [aws_security_group.cms_lb.id]
+  internal        = false
+}
+
+resource "aws_alb_target_group" "cms" {
+  name        = "cms-target-group"
+  port        = var.cms_app_port
+  protocol    = "HTTP"
+  vpc_id      = module.vpc.vpc_id
+  target_type = "ip"
+
+  health_check {
+    healthy_threshold   = "3"
+    interval            = "30"
+    protocol            = "HTTP"
+    matcher             = "200"
+    timeout             = "3"
+    path                = "/"
```

What about using [`_health` endpoint](https://forum.strapi.io/t/how-to-know-if-strapi-server-is-ready/3918/1)?

```suggestion
    matcher             = "204"
    timeout             = "3"
    path                = "/_health"
```